src/org/labkey/test/pages/LabkeyErrorPage.java

```diff
@@ -51,6 +57,15 @@ protected ElementCache newElementCache()
         return new ElementCache();
     }
 
+    public void assertUnauthorized(DeferredErrorCollector checker)
+    {
+        checker.verifyEquals("Incorrect error heading message", "Oops! An error has occurred.",
+                getErrorHeading());
+        checker.verifyEquals("Incorrect error sub-heading message", UNAUTHORIZED_FULL_PAGE_MESSAGE,
+                getSubErrorHeading());
+        checker.verifyThat("Incorrect error image", getErrorImage(), CoreMatchers.containsString(IMAGE_TITLE));
```

Comment:
Consider verifying the response code as well.
```suggestion
        checker.verifyThat("Incorrect error image", getErrorImage(), CoreMatchers.containsString(IMAGE_TITLE));
        checker.verifyEquals("Incorrect response code", 403, getResponseCode());
```